src/Simplify_Call.cpp

```diff
@@ -125,28 +127,39 @@ Expr Simplify::visit(const Call *op, ExprInfo *bounds) {
             return a;
         }
 
-        const bool shift_left = op->is_intrinsic(Call::shift_left);
+        //const bool const_shift_left = op->is_intrinsic(Call::shift_left);
```

Comment:
Did you mean to leave this commented out? If so, add a line explaining why. If not, remove.

Reply:
Removed, my bad.